File: HW5_b/Part 2/Apple.cpp
#include "Apple.h"

Apple::Apple(): Fruit()
{
	name = "apple";
}

Apple::Apple(time_t r): Fruit(ripe), Food(name)
{
	name = "apple";
	ripe = r;
}

void Apple::prepare()
{
	cout << "Core the " << name << endl;
}

File: HW4/Part 2/Apple.java
import java.util.*;

public class Apple extends Fruit
{
	public Apple()
	{
		name = "apple";
	}
	
	public Apple(Date r)
	{
		name = "apple";
		ripe = r;
	}
	
	public void prepare()
	{
		System.out.println("Core the " + name);
	}
}
